source/import-pipeline-from-text.txt

```diff
@@ -12,30 +12,58 @@ Import Pipeline from Text
    :depth: 1
    :class: singlecol
 
-*New in version 1.15.0*
-
 You can import aggregation pipelines from plain text into
 the :ref:`Aggregation Pipeline Builder <compass-agg-builder>` to
 easily modify and execute your pipelines. Importing a plain text
 aggregation pipeline shows how each stage of the pipeline
 affects the output, and illustrates the effects of modifying specific
 pipeline stages using the Pipeline Builder's :guilabel:`Output` panes.
 
+Starting in Compass 1.35, the import pipeline button has been removed
```

Comment:
Perhaps a `.. note::` admonition would be good here, as this kind of feels like an aside to me.